méli-mélo/2024-02-charts/meta.md

```diff
@@ -0,0 +1,44 @@
+---
+feature: méli-mélo
+"@context":
+  "@version": 1.0
+  dct: http://purl.org/dc/terms/
+  title:
+    "@id": dct:title
+    "@container": "@language"
+  description:
+    "@id": dct:description
+    "@container": "@language"
+  modified: dct:modified
+
+title:
+  en: Charts
+description:
+  en: Create a visual representation of data in a table.
+modified: 2024-02-26
+componentName: 2023-10-charts
+sponsor: Principal Publisher - ESDC
+
+pages:
+  examples:
+    - title: Charts - Working examples
+      language: en
+      path: demo.html
+  documentation:
+    - title: Charts - Documentation
+      language: en
+      path: index.html
+
+implementationPlan:
```

Comment:
This plan don't seem to be aligned with our guidance here: https://wet-boew.github.io/GCWeb/m%C3%A9li-m%C3%A9lo/m%C3%A9li-m%C3%A9lo-en.html#implementation-plan

We will need to include a few items, like engaging with DTO, producing WCAG assessment/ACR....